<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectTests</class>
 <widget class="QDialog" name="SelectTests">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1222</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Блок СКИ</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QWidget" name="pressure_1_color" native="true">
              <property name="minimumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #2A689F;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_box_pressure_1">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Давление 1</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QWidget" name="pressure_2_color" native="true">
              <property name="minimumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #457448;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_box_pressure_2">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Давление 2</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QWidget" name="pressure_3_color" native="true">
              <property name="minimumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #D3BB2A;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_box_pressure_3">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Давление 3</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QWidget" name="moving_color" native="true">
              <property name="minimumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #CC9546;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_box_moving">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Перемещение</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QWidget" name="output_4_20_mA_color" native="true">
              <property name="minimumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #A83240;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_box_output_4_20_mA">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Выход 4-20 mA</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QWidget" name="input_4_20_mA_color" native="true">
              <property name="minimumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #A83240;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_box_input_4_20_mA">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Вход 4-20 mA</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QWidget" name="usb_color" native="true">
              <property name="minimumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #25262B;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_box_usb">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="tabletTracking">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>USB</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QWidget" name="imit_switch_0_3_color" native="true">
              <property name="minimumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #5827AF;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_box_imit_switch_0_3">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Концевой выключатель 0-3</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <property name="autoRepeat">
               <bool>false</bool>
              </property>
              <property name="autoExclusive">
               <bool>false</bool>
              </property>
              <property name="tristate">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QWidget" name="imit_switch_3_0_color" native="true">
              <property name="minimumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #5827AF;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_box_imit_switch_3_0">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Концевой выключатель 3-0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QWidget" name="do_1_color" native="true">
              <property name="minimumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #25262B;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_box_do_1">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>DO 1</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QWidget" name="do_2_color" native="true">
              <property name="minimumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #25262B;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_box_do_2">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>DO 2</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QWidget" name="do_3_color" native="true">
              <property name="minimumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #25262B;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_box_do_3">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>DO 3</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QWidget" name="do_4_color" native="true">
              <property name="minimumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #25262B;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_box_do_4">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>DO 4</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Выбор программы испытаний</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QToolButton" name="button_CTV">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Комплексные испытания Отсечной Арматуры</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="button_BTSV">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Базовые испытания Запорно-Регулирующей Арматуры</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="button_CTSV">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Комплексные испытания Запорно-Регулирующей Армитуры</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="button_BTCV">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Базовые испытания Регулирующей Арматуры</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="button_CTCV">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Комплексные испытания Регулирующей Армитуры</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="entry_testing">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Далее</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
